examples/asgilook/asgilook/cache.py

```diff
@@ -0,0 +1,55 @@
+import msgpack
+
+
+class RedisCache:
+    PREFIX = 'asgilook:'
+    INVALIDATE_ON = frozenset({'DELETE', 'POST', 'PUT'})
+    CACHE_HEADER = 'X-ASGILook-Cache'
+    TTL = 3600
+
+    def __init__(self, config):
+        self.config = config
```

Comment:
Nitpick: Does this need an underscore prefix? Seems like a number of instance variables throughout the tutorial should actually be private.

Reply:
Good point.
I would normally make `self._config` private in similar code, I don't remember why it isn't the case here. :slightly_smiling_face: